apps/emqx_bridge_clickhouse/src/emqx_bridge_clickhouse_connector.erl

```diff
@@ -110,7 +110,7 @@
     %%     `VALUES [([<VALUE> | <PLACEHOLDER>], ...)]`
     %% Keep Capturing-Group:
     %%     `([<VALUE> | <PLACEHOLDER>], ...) [, ([<VALUE> | <PLACEHOLDER>], ..)]`
-    "(?:values\\s*(\\((?:[^()]++|(?2))*\\)(?:\\s*,\\s*\\((?:[^()]++|(?2)*)\\))*))",
+    "(?:values\\s*(\\((?:[^()]++|(?2))*\\)(?:\\s*,\\s*\\((?:[^()]++|(?2)*)\\))*)\\s*;?\\s*)",
```

Comment:
Q: Why is it not just part of `End Ignored Group`?

Reply:
it does, the real captured group is the part after `VALUES`
`VALUES` and the suffix `;` are not captured.
tested by
https://regex101.com/r/LDJaWs/3